bin/build-emcpp.sh

```diff
@@ -1,9 +1,8 @@
 #!/bin/sh
 
-#EMCC_DEBUG=1 
+#EMCC_DEBUG=1
 
-mkdir -p build && 
-cd build && 
-emconfigure cmake .. && 
-emmake cmake --build . -j `nproc` && 
-emmake make . -j `nproc`
+mkdir -p build &&
+cd build &&
+emconfigure cmake .. &&
```

Comment:
```suggestion
emcmake cmake .. &&
```

I think it's available in 1.38.48 and produces equivalent results, but `emconfigure cmake` is deprecated in newer emscripten version, so why not change while we are here.